pkg/boot/pe.go

```diff
@@ -78,7 +79,7 @@ func (p *PEImage) Execute() error {
 		entry = uintptr(oh.AddressOfEntryPoint)
 	}
 
-	if err := kexec.Load(M16+entry, segment, 0); err != nil {
+	if err := kexec.Load(M16+entry, segment, KEXEC_RUN_PE); err != nil {
```

Comment:
Sorry for reporting a bug here. In my case,My os is win10 enterprise version, bootmgfw.efi's   info as below:
AddressOfEntryPoint	0000000000014790
BaseOfCode		        0000000000001000
ImageBase		        0000000010000000
So, the start address should be 0x10014790, but the log is as below:
2019/12/26 06:51:32 virt: 0x7f95fff39000 + 0x400 | phys: 0x1000000 + 0x1000

2019/12/26 06:51:32 virt: 0x7f95fff39400 + 0xfe600 | phys: 0x1001000 + 0xfe544

2019/12/26 06:51:32 virt: 0x7f95fff39000 + 0x0 | phys: 0x1100000 + 0xde4

2019/12/26 06:51:32 virt: 0x7f9600037a00 + 0x800 | phys: 0x1101000 + 0x3a700

2019/12/26 06:51:32 virt: 0x7f9600038200 + 0x8400 | phys: 0x113c000 + 0x8340

2019/12/26 06:51:32 virt: 0x7f9600040600 + 0x10000 | phys: 0x1145000 + 0xfec0

2019/12/26 06:51:32 virt: 0x7f9600050600 + 0xa00 | phys: 0x1155000 + 0x820

2019/12/26 06:51:32 virt: 0x7f95fff39000 + 0x400 | phys: 0x1000000 + 0x1000

2019/12/26 06:51:32 virt: 0x7f95fff39400 + 0xfe600 | phys: 0x1001000 + 0xff000

2019/12/26 06:51:32 virt: 0x7f95fff39000 + 0x0 | phys: 0x1100000 + 0x1000

2019/12/26 06:51:32 virt: 0x7f9600037a00 + 0x800 | phys: 0x1101000 + 0x3b000

2019/12/26 06:51:32 virt: 0x7f9600038200 + 0x8400 | phys: 0x113c000 + 0x9000

2019/12/26 06:51:32 virt: 0x7f9600040600 + 0x10000 | phys: 0x1145000 + 0x10000

2019/12/26 06:51:32 virt: 0x7f9600050600 + 0xa00 | phys: 0x1155000 + 0x1000

2019/12/26 06:51:32 between adjust and dedup

2019/12/26 06:51:32 virt: 0x7f95fff39000 + 0x400 | phys: 0x1000000 + 0x1000

2019/12/26 06:51:32 virt: 0x7f95fff39400 + 0xfe600 | phys: 0x1001000 + 0xff000

2019/12/26 06:51:32 virt: 0x7f95fff39000 + 0x0 | phys: 0x1100000 + 0x1000

2019/12/26 06:51:32 virt: 0x7f9600037a00 + 0x800 | phys: 0x1101000 + 0x3b000

2019/12/26 06:51:32 virt: 0x7f9600038200 + 0x8400 | phys: 0x113c000 + 0x9000

2019/12/26 06:51:32 virt: 0x7f9600040600 + 0x10000 | phys: 0x1145000 + 0x10000

2019/12/26 06:51:32 virt: 0x7f9600050600 + 0xa00 | phys: 0x1155000 + 0x1000

2019/12/26 06:51:32 entry point: 0x1014790

### efi_hook_AllocatePages:1857; Num pages = 342; Allocation type: AllocateAddress; Memory type: EfiLoaderCode; Requested address = 0x10000000
### efi_hook_AllocatePages:1877; Allocated ffff888010000000 --> 0x10000000
### efi_setup_11_mapping_physical_addr:1265; start = 0x10000000, end = 0x10156000, vma->vm_start = 0xc000000000; vma->vm_end = 0xc004000000
......
The bug may be in  u-root-kexec_test\pkg\kexec\kexec_linux_amd64.go's func load.
Thank you for your attention.